app/models/order_detail.rb

```diff
@@ -718,8 +718,7 @@ def description
   end
 
   def long_description
-    "##{self}: #{order.user}: #{I18n.l(fulfilled_at.to_date, format: :usa)}: "\
-    "#{product} x#{quantity}"
+    OrderDetailJournalDescriptionPresenter.new(self).long_description
```

Comment:
It seems a little overkill to me to create a presenter just for one method--can you explain why you made that choice? Do you think other methods will eventually be added on that presenter?

Reply:
`long_description` is only used in journals. It's not really intrinsic to the OrderDetail. I would have gotten rid of `OrderDetail#long_description` and used the presenter in the journal, but the downstream forks's engines also rely on this method. So this was kind of a middle ground.